freetextresponse/templates/freetextresponse_view.html

```diff
@@ -30,15 +30,19 @@ <h2 class="problem-header">{{ display_name }}</h2>
     </div>
     <div class="capa_alert user_alert">{{ user_alert }}</div>
     {% if display_other_responses %}
-        <div class="responses-box">
+        <div class="responses-box {% if not student_answer %}hidden{% endif %}">
             <button class="hide-button">
                 <span class="hide">Hide</span>
                 <span class="show">Show</span>
                 <span class="sr">peer responses</span>
             </button>
             <p class="responses-title">Submissions by others</p>
             <ul class="response-list">
-                <li class="no-response">No responses to show at this time</li>
+                {% for response in other_responses %}
+                    <li class="other-student-responses">{{ response.answer }}</li>
+                {% empty %}
+                    <li class="no-response">No responses to show at this time</li>
```

Comment:
Maybe always write this item out to the template, `display: none;` by default, like before,
then display it when when `len(other_responses) == 0`.

Reply:
Sorry, you'd want to write it out so that you can grab it elsewhere in your javascript..